backgroundcheck_boilerplate/README.md

```diff
@@ -0,0 +1,18 @@
+# Hello World
+
+This is the default project that is scaffolded out when you run `npx @temporalio/create@latest ./myfolder`.
+
+The [Hello World Tutorial](https://learn.temporal.io/getting_started/typescript/hello_world_in_typescript/) walks through the code in this sample.
```

Comment:
I'm just confused why we have Hello World stuff in the README for background check? Thanks :)